### DISCUSSION TITLE
dshot beeper for when navigation ready?

### DISCUSSION THREAD

**Discussion by P-I-Engineer** [iNavFlight/inav#7140] [Q&A]
is there a way for the dshot beeper to beep when you have your 8 sats and are ready to fly?

on my quad with a beeper, it beeps when it's ready.

my plane using dshot beeper does not. Is there a setting i'm missing?

thanks

**Comment by DzikuVx** [comment]
Last time I checked, DSHTOT beeper was on after acquiring sats.
